[Microsoft 365 Lighthouse] - My roles feature

idwilliams opened the pull request:
Through this pull request the documentation for the new my roles feature is being added.

opbld31 commented:
Docs Build status updates of commit _[a97b58f](https://github.com/microsoftgraph/microsoft-graph-docs/commits/a97b58fc5d150e88ff900eed853387f47d4f8243)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-d748c732)

<a id="d748c732"></a>
###
 - **[Error: RuningBuildFailed]** `````Some unexpected errors happened when running build, please open a ticket in https://aka.ms/SiteHelp and include the error report for our team to troubleshoot`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C4%5C22%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202204221749156577-16372%5Cworkflow_report.html?sv=2021-04-10&se=2022-05-23T17%3A49%3A52Z&sr=b&sp=r&sig=FdWFLYX%2FXOxFLwd0GxZAdGfJO8jh6eTiKH50JmZ1JmE%3D).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

JarbasHorst commented on api-reference/beta/resources/managedtenants-roleassignment.md:
```diff
@@ -0,0 +1,44 @@
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|assignmentType|delegatedPrivilegeStatus|The type of admin relationship(s) associated with the role assignment. The possible values are: `none`, `delegatedAdminPrivileges`, `unknownFutureValue`, `granularDelegatedAdminPrivileges`, `delegatedAndGranularDelegetedAdminPrivileges`. Note that you must use the `Prefer: include - unknown -enum-members` request header to get the following value(s) in thi ...
```
If you will fix the typo in "delegatedAndGranular**Delegeted**AdminPrivileges", please update this docs entry here also.

idwilliams replied:
We are planning to update the spelling in the near feature, that way we can update the documentation and schema at the same time.

JarbasHorst commented on api-reference/beta/resources/managedtenants-roledefinition.md:
```diff
@@ -0,0 +1,42 @@
+|:---|:---|:---|
+|description|String|The description for the role.|
+|displayName|String|The display name for the role assignment.|
+|templateId|String|The well know unique identifier for the role.|
```
For simplicity, I recommend you to remove the word "well known" from this description.

idwilliams replied:
That is a great catch, this update will be published shortly.

JarbasHorst commented on changelog/Microsoft.MultiTenantPlatform.json:
```diff
@@ -1,49 +1,90 @@
-        "ChangeList": [
+  "changelog": [
+    {
+      "ChangeList": [
```
Please reflect in this changelog record the changes applied to the delegatedPrivilegeStatus enum:

- Added the \`granularDelegatedAdminPrivileges\` member to the \*\*delegatedPrivilegeStatus\*\* enumeration.
- Added the \`delegatedAndGranularDelegetedAdminPrivileges\` to the \*\*delegatedPrivilegeStatus\*\* enumeration.

idwilliams replied:
Completely missed those, thank you for flagging them.

JarbasHorst commented on api-reference/beta/resources/managedtenants-roleassignment.md:
```diff
@@ -0,0 +1,44 @@
+|Property|Type|Description|
+|:---|:---|:---|
+|assignmentType|delegatedPrivilegeStatus|The type of admin relationship(s) associated with the role assignment. The possible values are: `none`, `delegatedAdminPrivileges`, `unknownFutureValue`, `granularDelegatedAdminPrivileges`, `delegatedAndGranularDelegetedAdminPrivileges`. Note that you must use the `Prefer: include - unknown -enum-members` request header to get the following value(s) in thi ...
+|roles|[microsoft.graph.managedTenants.roleDefinition](../resources/managedtenants-intune-roledefinition.md) collection|The collection of roles assigned.|
```
Please, fix the reference link for microsoft.graph.managedTenants.roleDefinition as the current link is broken.

idwilliams replied:
Thank you for catching the update has been made and published to this branch.

JarbasHorst commented:
Hello, I reviewed your PR and left some comments. Please, take a look when you will have time.

Also, to avoid merge conflicts, please merge PR [16372](https://github.com/microsoftgraph/microsoft-graph-docs/pull/16372) into your branch to include my fixes and before addressing my comments.

opbld33 commented:
Docs Build status updates of commit _[964e616](https://github.com/microsoftgraph/microsoft-graph-docs/commits/964e6169f43e4bf06811446b5b5f468a401b8a2c)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/managedtenants-managedtenant-list-myroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/managedtenants-managedtenant-list-myroles.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) | [Details](#user-content-d71a78415c9e608dd5fe44faeb09e665-a682ab6b)
[api-reference/beta/resources/managedtenants-myrole.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-myrole.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-myrole?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) | [Details](#user-content-e7f81ebdf0e9054c1279cb9a8b6cf721-a682ab6b)
[api-reference/beta/resources/managedtenants-roleassignment.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roleassignment.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roleassignment?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) | [Details](#user-content-8e81131c6ac550f73381bb81362ccc68-a682ab6b)
[api-reference/beta/resources/enums-managedtenants.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/enums-managedtenants.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums-managedtenants?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-managedtenant.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-managedtenant.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-managedtenant?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roledefinition.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roledefinition.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roledefinition?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[changelog/Microsoft.MultiTenantPlatform.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/users/iswillia/managed-tenants-myrole/changelog/Microsoft.MultiTenantPlatform.json) | :white_check_mark:Succeeded | |

<a id="d71a78415c9e608dd5fe44faeb09e665-a682ab6b"></a>
### [api-reference/beta/api/managedtenants-managedtenant-list-myroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/managedtenants-managedtenant-list-myroles.md)
 - **Line 49, Column 81**: **[Warning: file-not-found - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../resources/myrole.md'.`````

<a id="e7f81ebdf0e9054c1279cb9a8b6cf721-a682ab6b"></a>
### [api-reference/beta/resources/managedtenants-myrole.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-myrole.md)
 - **Line 21, Column 2**: **[Warning: file-not-found - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../api/managedtenants-myrole-list.md'.`````

<a id="8e81131c6ac550f73381bb81362ccc68-a682ab6b"></a>
### [api-reference/beta/resources/managedtenants-roleassignment.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roleassignment.md)
 - **Line 22, Column 8**: **[Warning: file-not-found - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../resources/managedtenants-intune-roledefinition.md'.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C5%5C3%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202205030213189083-16372%5Cworkflow_report.html?sv=2021-04-10&se=2022-06-03T02%3A16%3A23Z&sr=b&sp=r&sig=NfgQB1gvIE9cbdFEn5DPJUUZJi2fYl3i5qJ%2F4sLTd%2Bc%3D).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

idwilliams commented on api-reference/beta/api/managedtenants-managedtenant-list-myroles.md:
```diff
@@ -0,0 +1,151 @@
+---
+title: "List myRoles"
+description: "Get the myRole resources from the myRoles navigation property."
```
This change has been made locally and will be published shortly.

idwilliams commented on api-reference/beta/resources/managedtenants-myrole.md:
```diff
@@ -0,0 +1,51 @@
+## Methods
+|Method|Return type|Description|
+|:---|:---|:---|
+|[List myRoles](../api/managedtenants-myrole-list.md)|[microsoft.graph.managedTenants.myRole](../resources/managedtenants-myrole.md) collection|Get a list of the [myRole](../resources/managedtenants-myrole.md) objects and their properties.|
```
This change has been made locally and will be published soon.

idwilliams commented on api-reference/beta/resources/enums-managedtenants.md:
```diff
@@ -16,6 +16,8 @@ doc_type: enumTypes
 |delegatedAdminPrivileges|
 |unknownFutureValue|
+|granularDelegatedAdminPrivileges|
+|delegatedAndGranularDelegetedAdminPrivileges|
```
Since this is a typo in the schema, our team has decided to update the spelling as part of a separate request. So, let's leave it misspelled for now.

JarbasHorst replied:
Thanks for the clarification.

idwilliams commented on api-reference/beta/api/managedtenants-managedtenant-list-myroles.md:
```diff
@@ -0,0 +1,151 @@
+
+## Response
+
+If successful, this method returns a `200 OK` response code and a collection of [myRole](../resources/myrole.md) objects in the response body.
```
This has been addressed locally and will be published soon.

idwilliams commented on api-reference/beta/resources/managedtenants-roleassignment.md:
```diff
@@ -0,0 +1,44 @@
+---
+title: "roleAssignment resource type"
+description: "Represent the role assignments you have."
```
These changes have been made locally and will be published shortly.

idwilliams commented on api-reference/beta/resources/enums-managedtenants.md:
```diff
@@ -16,6 +16,8 @@ doc_type: enumTypes
 |none|
 |delegatedAdminPrivileges|
 |unknownFutureValue|
```
When the schema was updated the position for the `unknownFutureValue` member was not moved to the bottom. So, we were trying to keep the order matching between the documentation and schema. Would you recommend that update the position in the documentation and not the schema?

JarbasHorst replied:
I have seen this now also in other PRs. It was a good idea to keep the order matching between the documentation and schema. No changes needed here.

idwilliams commented on api-reference/beta/api/managedtenants-managedtenant-list-myroles.md:
```diff
@@ -0,0 +1,151 @@
+          ]
+        },
+        {
+          "assignmentType": "granularDelegatedAccessPrivelege",
```
Thank you for catching this, updates have been locally and will be published shortly.

opbld34 commented:
Docs Build status updates of commit _[88d96ef](https://github.com/microsoftgraph/microsoft-graph-docs/commits/88d96efd95e2fb80b116cc1a61dc5b22a38da4cc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/managedtenants-managedtenant-list-myroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/managedtenants-managedtenant-list-myroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/enums-managedtenants.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/enums-managedtenants.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums-managedtenants?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-managedtenant.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-managedtenant.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-managedtenant?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-myrole.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-myrole.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-myrole?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roleassignment.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roleassignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roleassignment?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roledefinition.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roledefinition.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roledefinition?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[changelog/Microsoft.MultiTenantPlatform.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/users/iswillia/managed-tenants-myrole/changelog/Microsoft.MultiTenantPlatform.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C5%5C5%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202205052234442656-16372%5Cworkflow_report.html?sv=2021-04-10&se=2022-06-05T22%3A36%3A54Z&sr=b&sp=r&sig=Ngj6BNeIcqTjj2CUGkTQ0gjMY1ZEG%2BM6AsWOwtbNu24%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld34 commented:
Docs Build status updates of commit _[593e88c](https://github.com/microsoftgraph/microsoft-graph-docs/commits/593e88cc6f6a08f6ce081e2ab9933434a1db679c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/managedtenants-managedtenant-list-myroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/managedtenants-managedtenant-list-myroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/enums-managedtenants.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/enums-managedtenants.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums-managedtenants?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-managedtenant.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-managedtenant.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-managedtenant?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-myrole.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-myrole.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-myrole?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roleassignment.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roleassignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roleassignment?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roledefinition.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roledefinition.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roledefinition?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[changelog/Microsoft.MultiTenantPlatform.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/users/iswillia/managed-tenants-myrole/changelog/Microsoft.MultiTenantPlatform.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C5%5C5%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202205052239058416-16372%5Cworkflow_report.html?sv=2021-04-10&se=2022-06-05T22%3A41%3A16Z&sr=b&sp=r&sig=YbU%2FxCIwx51NfdOoFOGTUGOuY80233a3hDyVjYbQgcg%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld34 commented:
Docs Build status updates of commit _[9a4eb85](https://github.com/microsoftgraph/microsoft-graph-docs/commits/9a4eb8537ef41255b110752ff19d1decc6c3f6b5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/managedtenants-managedtenant-list-myroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/managedtenants-managedtenant-list-myroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/enums-managedtenants.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/enums-managedtenants.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums-managedtenants?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-managedtenant.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-managedtenant.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-managedtenant?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-myrole.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-myrole.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-myrole?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roleassignment.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roleassignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roleassignment?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roledefinition.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roledefinition.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roledefinition?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[changelog/Microsoft.MultiTenantPlatform.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/users/iswillia/managed-tenants-myrole/changelog/Microsoft.MultiTenantPlatform.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C5%5C5%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202205052253427802-16372%5Cworkflow_report.html?sv=2021-04-10&se=2022-06-05T22%3A55%3A47Z&sr=b&sp=r&sig=Z25%2F1vgQZpCR7k1UosT%2BLzCFGguDAPPbuX0wEdG75ok%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld31 commented:
Docs Build status updates of commit _[df5533c](https://github.com/microsoftgraph/microsoft-graph-docs/commits/df5533caf31f113df1814b97a57c6077b6c54096)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/managedtenants-managedtenant-list-myroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/managedtenants-managedtenant-list-myroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/enums-managedtenants.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/enums-managedtenants.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums-managedtenants?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-managedtenant.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-managedtenant.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-managedtenant?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-myrole.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-myrole.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-myrole?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roleassignment.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roleassignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roleassignment?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roledefinition.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roledefinition.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roledefinition?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[changelog/Microsoft.MultiTenantPlatform.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/users/iswillia/managed-tenants-myrole/changelog/Microsoft.MultiTenantPlatform.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C5%5C6%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202205061654244522-16372%5Cworkflow_report.html?sv=2021-04-10&se=2022-06-06T16%3A57%3A42Z&sr=b&sp=r&sig=xNJ8b8ikUZskjZDCqnTTIFEj067xUY9LeJSf9QIskdY%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

JarbasHorst commented:
@Lauragra this PR is ready for final review. Please, merge PR [16839](https://github.com/microsoftgraph/microsoft-graph-docs/pull/16839) into this branch to include my final edits.

@idwilliams, for clarity, I changed some descriptions to include "a signed in user" instead of you.

opbld33 commented:
Docs Build status updates of commit _[2f33520](https://github.com/microsoftgraph/microsoft-graph-docs/commits/2f33520e560dee7e5a7db0f835c004a0de30c967)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/managedtenants-managedtenant-list-myroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/managedtenants-managedtenant-list-myroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/enums-managedtenants.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/enums-managedtenants.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums-managedtenants?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-managedtenant.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-managedtenant.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-managedtenant?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-myrole.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-myrole.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-myrole?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roleassignment.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roleassignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roleassignment?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roledefinition.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roledefinition.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roledefinition?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[changelog/Microsoft.MultiTenantPlatform.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/users/iswillia/managed-tenants-myrole/changelog/Microsoft.MultiTenantPlatform.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/77e68b91-cf14-3558-43ca-0d6d735c492d/202205110148415475-16372/BuildReport?accessString=f7367ea9fac0247e42222e55abbf5ca655c267b5918ca0ae4d4dcea71a86894d).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

idwilliams commented:
Updated the changelog to resolve the merge conflicts. Also, it appears the date was missing from the latest addition to the changelog. I have added that property and adjusted the date to be more recent.

opbld34 commented:
Docs Build status updates of commit _[5de2b9d](https://github.com/microsoftgraph/microsoft-graph-docs/commits/5de2b9d366adb26f2a19f59b51d64cac7a771b3f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/managedtenants-managedtenant-list-myroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/managedtenants-managedtenant-list-myroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/enums-managedtenants.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/enums-managedtenants.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums-managedtenants?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-managedtenant.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-managedtenant.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-managedtenant?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-myrole.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-myrole.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-myrole?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roleassignment.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roleassignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roleassignment?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roledefinition.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roledefinition.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roledefinition?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[changelog/Microsoft.MultiTenantPlatform.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/users/iswillia/managed-tenants-myrole/changelog/Microsoft.MultiTenantPlatform.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/77e68b91-cf14-3558-43ca-0d6d735c492d/202205131206249038-16372/BuildReport?accessString=eca2170cc6b22ee5c219e3e9878207ee390e9285ab65e3bd0dca33980093dec9).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld30 commented:
Docs Build status updates of commit _[0f38ca5](https://github.com/microsoftgraph/microsoft-graph-docs/commits/0f38ca5dedfcc731eb27b0cd7357bc0386975663)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/managedtenants-managedtenant-list-myroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/managedtenants-managedtenant-list-myroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/enums-managedtenants.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/enums-managedtenants.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums-managedtenants?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-managedtenant.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-managedtenant.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-managedtenant?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-myrole.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-myrole.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-myrole?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roleassignment.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roleassignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roleassignment?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roledefinition.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roledefinition.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roledefinition?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[changelog/Microsoft.MultiTenantPlatform.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/users/iswillia/managed-tenants-myrole/changelog/Microsoft.MultiTenantPlatform.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/77e68b91-cf14-3558-43ca-0d6d735c492d/202206030420153615-16372/BuildReport?accessString=bb94b48634b5f8d6b0c40332b9985c6ce73ba902b88150372fe50ee505e90979).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld32 commented:
Docs Build status updates of commit _[17c4266](https://github.com/microsoftgraph/microsoft-graph-docs/commits/17c4266978b5ae7aa3d3bae521f283f103d6b58c)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-f8770ce9)

<a id="f8770ce9"></a>
###
 - **[Error: PullRequestMergeConflict]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/77e68b91-cf14-3558-43ca-0d6d735c492d/202206140157053937-16372/BuildReport?accessString=3f932fd32e7acfccfc66569a06f1fb7d1b5e2456a974fa5be5257f968c3c3af8).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld31 commented:
Docs Build status updates of commit _[bda1c75](https://github.com/microsoftgraph/microsoft-graph-docs/commits/bda1c753ece1e2574bf204b0f9e651c8ce626599)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/managedtenants-managedtenant-list-myroles.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/managedtenants-managedtenant-list-myroles.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/enums-managedtenants.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/enums-managedtenants.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums-managedtenants?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-managedtenant.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-managedtenant.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-managedtenant?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-myrole.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-myrole.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-myrole?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roleassignment.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roleassignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roleassignment?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/resources/managedtenants-roledefinition.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/managedtenants-roledefinition.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/managedtenants-roledefinition?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[api-reference/beta/toc.yml](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/managedtenants-managedtenant-list-myroles?view=graph-rest-beta&branch=pr-en-us-16372) (graph-rest-beta) |
[changelog/Microsoft.MultiTenantPlatform.json](https://github.com/microsoftgraph/microsoft-graph-docs/blob/users/iswillia/managed-tenants-myrole/changelog/Microsoft.MultiTenantPlatform.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/77e68b91-cf14-3558-43ca-0d6d735c492d/202206140159164488-16372/BuildReport?accessString=7db394bec3b489eb8db0b49ca83d08fc0997eeee2300ef428557503e4e4ff971).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
